src/Sulu/Bundle/MediaBundle/Resources/translations/sulu/backend.de.xlf

```diff
@@ -114,6 +114,18 @@
                 <source>sulu.media.edit.loading</source>
                 <target>Medien werden geladen</target>
             </trans-unit>
+            <trans-unit id="243d88cf2f64652633a5ca2169f22258" resname="sulu.media.history">
+                <source>sulu.media.history</source>
+                <target>History</target>
```

Comment:
@alexander-schranz Do we also use `History` as the german translation? /cc @berndhep